src/ScottPlot/StarAxis.cs

```diff
@@ -130,11 +142,23 @@ public void Render(PlotDimensions dims, Bitmap bmp, bool lowQuality = false)
                 }
             }
 
-            if (CategoryLabels != null && ShowCategoryLabels)
+            if (CategoryImages != null && ShowCategoryImages)
             {
                 for (int i = 0; i < NumberOfSpokes; i++)
                 {
+                    double cosinus = Math.Cos(sweepAngle * i - Math.PI / 2);
+                    double sinus = Math.Sin(sweepAngle * i - Math.PI / 2);
```

Comment:
Is that what I should have been calling this all along? It's a good variable name, but it does sound like the sinus cavity.

If you don't mind me asking, do you speak another language? I know that in French the functions are called sinus and cosinus.

Reply:
I speak spanish and I went with what I think is the most aproximate word, feel free to change to whichever suits better :)

Reply:
Oh cool, I'm Canadian so I have a teensie bit of French, but it's not great. I don't have a problem with the variable name, I just thought it was interesting.

Reply:
I'll update both names to `cosine` and `sine`, sometimes it's worth to check proper spelling of a particular word.

Reply:
I actually like the current name, it distances it from the sine/cosine function a little bit

Reply:
Then I'll leave them as cosinus and sinus :)